stable/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 9.1.1
+version: 9.1.2
```

Comment:
With a change as complex as the `apiVersion`, and the introduction of the *requirement* for Tiller via the `.Capabilities` call, I feel this should be more than a patch version change.

Reply:
9.2.0 ?

Reply:
@WarheadsSE I updated the version to 9.2.0.

Reply:
Minor version upgrade seems right to me for this. I'm unclear on the `tide` check, but looks like we need the `lgtm` to come from a maintainer..?

Reply:
@zaneclaes yes, it requires `/lgtm` comment from a maintainer.